zach_kaz/forloopbasic2.py

```diff
@@ -0,0 +1,70 @@
+def biggie(arr):
+    for i in arr:
+        if i>0:
+            arr[i]="big"
+    return arr
+
+def countpos(arr):
+    temp=0
+    for i in arr:
+        if arr[i]>0:
+            temp+=1
+    arr[len(arr)-1]=temp
+    return arr
+
+def sumtotal(arr):
+    x=0
+    for i in arr:
+        x+=i
+    return x
+
+def average(arr):
+    x=0
+    for i in arr:
+        x+=1
+    return x/len(arr)
+
+def length(arr)
+    x=0
+    for i in arr:
+        x+=1
+    return x
+
+def minimum(arr)
+    if arr==[]:
+        return false
+    x=arr[0]
+    for i in arr:
+        if x>arr[i]:
+            x=arr[i]
+    return x
+
+def maximum(arr)
+    if arr==[]:
+        return false
+    x=arr[0]
+    for i in arr:
+        if x<arr[i]:
+            x=arr[i]
+    return x
+
+def ultimateAnalyze(arr):
+    dic={sumtotal:0,average:0,minimum:arr[0],maximum:arr[0],length:0}
```

Comment:
I believe keys should be in quotes
d = {'sumtotal' : 0}
Also if the array is empty arr[0] does not exist